src/transform/FooterReadMore.css

```diff
@@ -0,0 +1,92 @@
+
+.pagelib_footer_readmore_page {
+  background-color: white;
+  padding: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  padding-top: 11px;
```

Comment:
Consider folding this into `padding` a couple lines up as `padding: 11px 10px;`.

https://developer.mozilla.org/en-US/docs/Web/CSS/padding#Syntax